cmake/SSGCommon.cmake

```diff
@@ -522,7 +522,7 @@ macro(ssg_build_sds PRODUCT)
         set_tests_properties("missing-references-ssg-${PRODUCT}-ds.xml" PROPERTIES LABELS quick)
 
     endif()
-    if(("${PRODUCT}" MATCHES "ubuntu2004" OR "${PRODUCT}" MATCHES "rhel8") AND SSG_SCE_ENABLED)
```

Comment:
The removal of this can cause some issues when building with SCE enabled, and you have products built without SCE content. I don't agree we can remove this in the current state. 

```
$ export ADDITIONAL_CMAKE_OPTIONS="-DSSG_SCE_ENABLED:BOOL=ON"
$ ./build_product rhel7
$ cd build && ctest -E linkchecker --output-on-failure

99% tests passed, 1 tests failed out of 71

Label Time Summary:
quick    = 126.88 sec*proc (42 tests)

Total Test time (real) = 120.70 sec

The following tests FAILED:
	 71 - ds-sce-rhel7 (Failed)
Errors while running CTest
```

Reply:
I was talking to @marcusburghardt about it, I still feel we could remove that match of products and instead change the test to not exit(1) if SCE checks are not found in the datastream.
https://github.com/ComplianceAsCode/content/blob/master/tests/test_ds_sce.py#L77

Thougts?

Reply:
That's a solution. However, if somehow build system no longer adds SCE this check will always pass. Maybe we can cover that some other way.

Reply:
or maybe should the SSG_SCE_ENABLED be a cmake config in the products/*/CMakeLists.txt somehow, not sure how it would fit with current build system

Reply:
That would require some reworking of the build system as currently `SSG_SCE_ENABLED` is more of a global setting.

Reply:
ok, I've moved away from removing that line for now, might come back to it in the future.